components/layout_2020/style_ext.rs

```diff
@@ -671,7 +676,13 @@ impl ComputedValuesExt for ComputedValues {
             return true;
         }
 
-        if !self.get_effects().filter.0.is_empty() {
+        let effects = self.get_effects();
+
+        if !effects.filter.0.is_empty() {
+            return true;
+        }
+
+        if !effects.backdrop_filter.0.is_empty() {
```

Comment:
Please add a FIXME while you are at it:

```suggestion
        // FIXME(#35391): `filter` and `backdrop-filter` shouldn't establish
        // a containing block when applied to the document root element.
        let effects = self.get_effects();
        if !effects.filter.0.is_empty() || !effects.backdrop_filter.0.is_empty() {
            return true;
        }
```